src/app/frontend/logs/logstoolbar/logstoolbar_controller.js

```diff
@@ -130,7 +130,14 @@ export default class LogsToolbarController {
    * @return {!backendApi.ReplicationControllerPodWithContainers|undefined}
    * @private
    */
-  findPodByName_(array, name) { return array.find((element) => element.name === name); }
+  findPodByName_(array, name) {
```

Comment:
Can we maybe extend array object prototype with this functions manually? It would allow use to use it in the old way.